packages/phasor/src/elements/debug/debug-element.ts

```diff
@@ -8,13 +8,15 @@ export interface IDebug {
   xywh: SerializedXYWH;
   index: string;
   seed: number;
+  rotate: number;
 
   color: string;
 }
 
 export const DebugElementDefaultProps: IElementDefaultProps<'debug'> = {
   type: 'debug',
   xywh: '[0,0,0,0]',
+  rotate: 0,
```

Comment:
I think using `xywhr` that packs rotate value may be a better choice.

Reply:
Well, we can add rotate at the end.

Reply:
I think we can consider adding the `matrix`, `width` `height` fields.